utils/freifunk-berlin-network-defaults/uci-defaults/freifunk-berlin-default-txpower

```diff
@@ -0,0 +1,33 @@
+#!/bin/ash
+#
+# On Ubiquiti it has been seen that the default TX-power on OpenWRT
+# is at setting of Hardware, which is above regulatory limits
+# see https://github.com/freifunk-berlin/firmware/issues/381
+#
+
+. /lib/functions/guard.sh
+guard "2GHz_txpower"
```

Comment:
"2GHz_txpower" sounds generic, any reason against making this something like "ubnt_nsm_24ghz_txpower"?